.gitattributes

```diff
@@ -1 +0,0 @@
-*.onnx filter=lfs diff=lfs merge=lfs -text
```

Comment:
Undo this change. We want .onnx files to be stored with git LFS. If CI was crashing because of it, the fix must be done on the CI config side.